data/pug.txt

```diff
@@ -317,6 +317,16 @@ outfit "Pug Biodefenses"
 	"unplunderable" 1
 	description "This defense system is designed to release a suite of toxins that are lethal to human beings (and probably to some other species, as well) but which have no effect at all on the Pug."
 
+outfit "Pug Peacekeeping Staff"
+	plural "Pug Peacekeeping Staves"
+	category "Hand to Hand"
+	cost 123456
+	thumbnail "outfit/pug staff"
+	"capture attack" 2.3
+	"capture defense" 2.3
+	"unplunderable" 1
+	description "Despite being designed for planetary population control, peacekeeping staves also work exceptionally well in ship defense. The staff emits a short-ranged beam that immediately incapacitates anyone it touches by causing them to enter a euphoric and hallucinogenic state, wandering off to somewhere where they won't cause any trouble. Exiting this euphoric state is never pleasant, though, often resulting in vomiting."
```

Comment:
`Despite being` could be shortened to `Though with no loss of meaning

`peacekeeping staves also work` -> `the peacekeeping staff also works` (otherwise it's referred to in the plural in one sentence, and in the singular in the next)

`well in` -> `well for`

`immediately` is probably not needed

`Exiting this euphoric state is never pleasant, though, often resulting in vomiting.`
->
`Coming down from the state of euphoria is never pleasant.` (or something like that; I don't think mentioning the vomiting is really necessary)